docs/en/maintain/monitoring.md

```diff
@@ -0,0 +1,200 @@
+      openmldb_connected_seconds_created{endpoint="172.17.0.15:9624",role="nameserver"} 1.6501813860512598e+09               
+      
+   ```
+
+## Deploy node exporter
+For how to install and deploy prometheus, grafana, please refer to the official documents [promtheus get started](https://prometheus.io/docs/prometheus/latest/getting_started/) and [grafana get started](https://grafana.com/docs/ grafana/latest/getting-started/getting-started-prometheus/) .
+
+[node_exporter](https://github.com/prometheus/node_exporter) is an official implementation of prometheus that exposes system metrics.
+
+Go to the [release](https://github.com/prometheus/node_exporter/releases) page, download and unzip the compressed package of the corresponding platform. For example, under linux amd64 platform:
+```sh
+curl -SLO https://github.com/prometheus/node_exporter/releases/download/v1.3.1/node_exporter-1.3.1.darwin-amd64.tar.gz
+tar xzf node_exporter-1.3.1-*.tar.gz
+cd node_exporter-1.3.1-*/
+
+# Start node_exporter
+./node_exporter
+```
+
+## Deploy Prometheus and Grafana
+
+For installation and deployment of prometheus and grafana, please refer to the official documents [promtheus get started](https://prometheus.io/docs/prometheus/latest/getting_started/) and [grafana get started](https://grafana.com/docs/ grafana/latest/getting-started/getting-started-prometheus/).
+OpenMLDB provides prometheus and grafana configuration files for reference, see [OpenMLDB mixin](https://github.com/4paradigm/OpenMLDB/tree/main/monitoring/openmldb_mixin/README.md)
+
+- prometheus_example.yml: Prometheus configuration example, intended to modify the target address in `node`, `openmldb_components` and `openmldb_exporter` jobs
+- openmldb_dashboard.json: grafana dashboard configuration for OpenMLDB metrics, divided into two steps:
+   1. Under the grafana data source page, add the started prometheus server address as the data source
+   2. Under the dashboard browsing page, click New to import a dashboard, and upload the json configuration file
+
+## Understand existing monitoring metrics
+
+Taking the OpenMLDB cluster system as an example, the monitoring indicators are divided into two categories according to different prometheus pull jobs:
+
+1. DB-Level metrics, exposed through the OpenMLDB exporter, correspond to the `job_name=openmldb_exporter` entry in the `prometheus_example.yml` configuration:
+
+   ```yaml
+     - job_name: openmldb_exporter
+       # pull OpenMLDB DB-Level specific metric
+       # change the 'targets' value to your deployed OpenMLDB exporter endpoint
+       static_configs:
+         - targets:
+           - 172.17.0.15:8000
+   ```
+
+   The categories of indicators exposed are mainly:
+
+   - component status: cluster component status
+
+   - table status: database table related information, such as `rows_cout`, `memory_bytes`
+
+   - deploy query response time: the runtime of the deployment query inside the tablet
+
+   accessible
```

Comment:
The full DB-Level metrics can be listed through the following command:

(sorry the Chinese doc is also not good)